mlflow/pytorch/__init__.py

```diff
@@ -343,24 +418,59 @@ def save_model(
     .. code-block:: python
         :caption: Example
 
+        import os
+
+        import numpy as np
         import torch
-        import mlflow
         import mlflow.pytorch
-        # Create model and set values
-        pytorch_model = Model()
-        pytorch_model_path = ...
-        # train our model
-        for epoch in range(500):
-            y_pred = pytorch_model(x_data)
-            ...
-        # Save the model
+
+        # Class defined here
+        class LinearNNModel(torch.nn.Module):
+        ...
+
+        # Initialize our model, criterion and optimizer
+        model = LinearNNModel()
+        ...
+
+        # Training loop
+        ...
+
+        # Save PyTorch models to current working directory
         with mlflow.start_run() as run:
-            mlflow.log_param("epochs", 500)
-            mlflow.pytorch.save_model(pytorch_model, pytorch_model_path)
+            mlflow.pytorch.save_model(model, "models_pth")
 
-            # Saving scripted model
+            # Convert to a scripted model and save it
             scripted_pytorch_model = torch.jit.script(model)
-            mlflow.pytorch.save_model(scripted_pytorch_model, pytorch_model_path)
+            mlflow.pytorch.save_model(scripted_pytorch_model, "scripted_models_pth")
+
+        # Load each saved model for inference
+        print("--")
+        for model_path in ["models_pth", "scripted_models_pth"]:
+            model_uri = "{}/{}".format(os.getcwd(), model_path)
```

Comment:
Is this line really correct? `"{}/{}".format(os.getcwd(), model_path)` doesn't seem to be the right path.

Reply:
Yes, it [seems to work.](https://github.com/dmatrix/mlflow-tests/blob/master/py/mlflow/models/pytorch/save_model.py) The saving model saves it under the local path.

Reply:
I think this is because the artifact directory points to the current working directory.

Reply:
Is there anything wrong with it If the current artifact directory points to cwd and it's where you wish to save the model?

Reply:
If the current artifact directory didn't point to cwd, it would fail. I think it's safe to use `mlflow.get_artifact_uri`.

Reply:
or we could use `runs:{}/{}` format.

Reply:
sorry I misunderstood this is `log_model`.